exotica_python/src/pyexotica.cpp

```diff
@@ -1109,9 +1109,22 @@ PYBIND11_MODULE(_pyexotica, module)
     collision_scene.def("update_collision_object_transforms", &CollisionScene::UpdateCollisionObjectTransforms);
     collision_scene.def("continuous_collision_check", &CollisionScene::ContinuousCollisionCheck);
 
-    py::class_<Visualization> visualization(module, "Visualization");
-    visualization.def(py::init<ScenePtr>());
-    visualization.def("display_trajectory", &Visualization::DisplayTrajectory);
+    py::class_<VisualizationMoveIt> visualization_moveit(module, "VisualizationMoveIt");
```

Comment:
Ok - this is for future (please open an issue), I'd like to get a general Visualization class going with DisplayState, DisplayTrajectory, DisplayScene - and throwing if overload not implemented. Then we can expose general Visualization and only the differing elements (get_url etc) for the implementations